pkg/rulefmt/rulefmt.go

```diff
@@ -34,48 +35,54 @@ type Error struct {
 	Rule     int
 	RuleName string
 	Err      error
+	Node     yaml.Node
 }
 
 func (err *Error) Error() string {
```

Comment:
Why are we wrapping an error, and then unwrapping it again by returning a string?

Reply:
errors returned by the parser need a `yaml.Node` wrapper as well. Hence for those cases, I am returning a Node which is then unwrapped to string at the time of return.